site/pages/gc-form-en.hbs

```diff
@@ -0,0 +1,197 @@
+---
+{
+	"title": "Checkboxes and radio buttons",
+	"language": "en",
+	"altLangPrefix": "gc-form",
+	"pageType": "gcform",
+	"pageclass": "page-type-nav cnt-wdth-lmtd",
+	"breadcrumb": [
+		{ "title": "Home", "link": "https://www.canada.ca/en.html" },
+	],
+	"dateModified": "2019-08-20",
+	"share": "true"
+}
+---
+<section>
+	<h2>Purpose</h2>
+	<p>Checkboxes and radio buttons with large touch targets to support behavioural patterns of interacting with the form element, and not the label.</p>
+</section>
+<section>
+	<h2>Usage</h2>
+	<div class="row wb-eqht mrgn-tp-lg mrgn-bttm-xl">
+		<div class="col-md-6">
+			<h3 class="mrgn-tp-0 text-success h4">Use when <span class="glyphicon glyphicon-ok-circle"></span></h3>
+			<ul>
+				<li>Streamline complex navigation through multiple option.</li>
+				<li>Use for interactive questions and checkboxes for services and wizards</li>
+			</ul>
+		</div>
+		<div class="col-md-6 brdr-lft">
+			<h3 class="mrgn-tp-0 text-danger h4">Do not use when <span class="glyphicon glyphicon-remove-circle"></span></h3>
+			<ul>
+				<li>Avoid using checkboxes or radio buttons if the options are not easily comparable. Consider navigation patterns such as a topic page</li>
+				<li>If supporting navigation, for a small number of options, consider using expand/collapse or tabs</li>
+			</ul>
+		</div>
+	</div>
+</section>
+<section>
+	<h2>Working examples</h2>
+	<h3>Checkboxes</h3>
+	<div class="row">
+		<div class="col-md-4">
+			<div class="panel panel-default">
+				<div class="panel-body">
+					<h5 class="mrgn-tp-0">Appearance</h5>
+					<h6>Large checkboxes</h6>
+					<ul class="list-unstyled gc-form">
+						<li class="checkbox">
+							<input type="checkbox" id="checkbox_1">
+							<label for="checkbox_1">Option 1</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="checkbox_2">
+							<label for="checkbox_2">Option 2</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="checkbox_3">
+							<label for="checkbox_3">Option 3</label>
+						</li>
+						<li class="checkbox">
+							<input type="checkbox" id="checkbox_4">
+							<label for="checkbox_4">Option 4</label>
+						</li>
+					</ul>
+					<hr>
+					<h6>Small checkboxes</h6>
+					<ul class="list-unstyled gc-form">
```

Comment:
Is the CSS class ```gc-form``` need to be set on the unordered list or it can be set on a parent form element? 

See my related comment https://github.com/wet-boew/GCWeb/pull/1584#pullrequestreview-283837546

Reply:
Should work on parent elements. I will check with Jennifer and add a description.